pkg/kubelet/dockertools/manager.go

```diff
@@ -1354,6 +1354,10 @@ func (dm *DockerManager) killContainer(containerID kubecontainer.ContainerID, co
 	if gracePeriod < minimumGracePeriodInSeconds {
 		gracePeriod = minimumGracePeriodInSeconds
 	}
+	if gracePeriodOverride != nil {
```

Comment:
btw, systemd defaults to 90s for default stop timeout, not sure why we settled on 2s here...

https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html